source/extensions/filters/http/ext_proc/config.cc

```diff
@@ -50,6 +50,18 @@ absl::Status verifyProcessingModeConfig(
         "then the response_trailer_mode has to be set to SEND");
   }
 
+  // Do not support fail open for FULL_DUPLEX_STREAMED body mode.
+  if ((processing_mode.request_body_mode() ==
+       envoy::extensions::filters::http::ext_proc::v3::ProcessingMode::FULL_DUPLEX_STREAMED) ||
+      (processing_mode.response_body_mode() ==
+       envoy::extensions::filters::http::ext_proc::v3::ProcessingMode::FULL_DUPLEX_STREAMED)) {
+    if (config.failure_mode_allow()) {
```

Comment:
nit can this if be merged to if condition above? as suggested here https://github.com/envoyproxy/envoy/pull/39740#discussion_r2127200109

Thanks!

Reply:
done